Developer-Guide.md

```diff
@@ -688,3 +691,23 @@ file exists and contains details of the image and how it was created:
 ```
 $ cat /var/lib/osbuilder/osbuilder.yaml
 ```
+
+## Running standalone
+
+It is possible to start the runtime without a container manager. This is
+mostly useful for testing and debugging purposes.
+
+### Create an OCI bundle
+
```

Comment:
The way github renders the sub-sections ('###') does not make it clear to my eyes that these two sections are steps that need to be taken. I think we could improve that either by:
- adding some verbage (even a single sentence) such as `Build an OCI bundle using the runtime and a docker export`)
- drop the '###' section titles and convert them into just bullets or statements

Reply:
OK - how does the branch look now to you?